src/lists.h

```diff
@@ -257,6 +257,8 @@ extern  Obj (*Elm0vListFuncs[LAST_REAL_TNUM+1]) ( Obj list, Int pos );
 */
 static inline Obj ELMV0_LIST(Obj list, Int pos)
 {
+    GAP_ASSERT(pos > 0);
+    GAP_ASSERT(pos <= LEN_LIST(list));
```

Comment:
Looks good to me... Although I do wonder why we pass the `pos` as an `Int`, instead of an `UInt`? Then we could get rid of the first assert, too... @ChrisJefferson any particular reason?

Reply:
Hm. The other inline list access functions all/mostly also use `Int`. That could also be changed if that happens.

Reply:
We still need to first assert, because we want to forbid 0.

For `Int` vs `UInt`, I tried to use `Int` where possible, because most of our code which calls these functions uses `Int`.